src/ExactAbstract.py

```
@@ -0,0 +1,37 @@
+from flask import Flask
+from flask import request
+from flask import render_template
+from algorithms.statistical import get_keyword
+from nltk.tokenize import word_tokenize
+import json
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+@app.route('/keywords', methods=['POST'])
+def get_keywords():
+    if request.method == 'POST':
+        text = request.form['text']
+        return json.dumps(get_keyword(text))
+    else:
+        return 'Something has gone terribly wrong.'
```

I agree with the message, but we should return standard HTTP error codes where it makes sense.  Flask has a nice set of standard black and white error pages for error codes (in this case 405 Method Not Allowed): `abort(405)` instead of the `return` statement.
This would also be a good place for some negative automated tests: check to see that `/keywords` only accepts a `POST` method